### PR TITLE
Add I_S.keywords for v7.5

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

The `information_schema.keywords` has been backported to v7.5 as MySQL Connector/J uses this table if the advertised MySQL version is v8.0 or higher. This makes it easier to use v7.5 with Connector/J without having to configure TiDB server to advertise a lower version of MySQL.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

- https://github.com/pingcap/tidb/pull/54219
- https://github.com/pingcap/tidb/pull/54220
- https://github.com/pingcap/tidb/pull/54221

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-keywords.md
Comment (Oreoxmt) [inline information-schema/information-schema-keywords.md:1]: Please add this file to TOC.md and TOC-tidb-cloud.md

File: information-schema/information-schema.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefi
 | `FILES`                                                                                 | Not implemented by TiDB. Returns zero rows. |
 | `GLOBAL_STATUS`                                                                         | Not implemented by TiDB. Returns zero rows. |
 | `GLOBAL_VARIABLES`                                                                      | Not implemented by TiDB. Returns zero rows. |
+| [`KEYWORDS`](/information-schema/information-schema-keywords.md)                        | Provides a full list of keywords.
```
Comment (Oreoxmt) [inline information-schema/information-schema.md:30]:
```suggestion
| [`KEYWORDS`](/information-schema/information-schema-keywords.md)                        | Provides a full list of keywords.                |
```

**Review by Oreoxmt — APPROVED** [review]

**Comment by Oreoxmt** [comment]
ref: https://github.com/pingcap/docs/pull/15794/files#r1434146500

@dveeden https://github.com/dveeden/docs/pull/5 PTAL

File: develop/dev-guide-choose-driver-or-orm.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ You can follow the [MySQL documentation](https://dev.mysql.com/doc/connector-j/e
...
-> - When you are using MySQL Connector/J 8.0 with TiDB v7.5.x, it is recommended to set the TiDB configuration item [`server-version`](https://docs.pingcap.com/tidb/v7.5/tidb-configuration-file#server-version) to `"5.7.25-TiDB-v7.5.x"`. MySQL Connector/J attempts to access the `information_schema.KEYWORDS` table if the TiDB server reports a version of MySQL 8.0.11 or later. However, this table is not present in TiDB v7.5.x.
+> - When you are using MySQL Connector/J 8.0 with TiDB v7.5.2 or earlier versions, it is recommended to set the TiDB configuration item [`server-version`](https://docs.pingcap.com/tidb/v7.5/tidb-configuration-file#server-version) to `"5.7.25-TiDB-v7.5.x"`. MySQL Connector/J attempts to access the `information_schema.KEYWORDS` table if the TiDB server reports a version of MySQL 8.0.11 or later. However, TiDB provides this table in v7.5.3.
```
Comment (dveeden) [inline develop/dev-guide-choose-driver-or-orm.md:35]:
```suggestion
> - When you are using MySQL Connector/J 8.0 with TiDB v7.5.2 or earlier versions, it is recommended to set the TiDB configuration item [`server-version`](https://docs.pingcap.com/tidb/v7.5/tidb-configuration-file#server-version) to `"5.7.25-TiDB-v7.5.x"`. MySQL Connector/J attempts to access the [`information_schema.KEYWORDS`](/information-schema/information-schema-keywords.md) table if the TiDB server reports a version of MySQL 8.0.11 or later. However, TiDB provides this table in v7.5.3.
```

**Comment by Oreoxmt** [comment]
@bb7133 PTAL

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**